lib/components/bottom_navbar_widget.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:pace_app/bottom_navbar/bloc/bottom_navbar_bloc.dart';
+import 'package:pace_app/bottom_navbar/bloc/bottom_navbar_event.dart';
+import 'package:pace_app/bottom_navbar/bloc/bottom_navbar_state.dart';
+
+class BottomNavbarWidget extends StatefulWidget {
+  @override
+  _BottomNavbarWidgetState createState() => _BottomNavbarWidgetState();
+}
+
+class _BottomNavbarWidgetState extends State<BottomNavbarWidget> {
+  int _selectedIndex = 0;
+
+  final List<BottomNavigationBarItem> _listItems = [
+    BottomNavigationBarItem(
+        label: 'Play', icon: Icon(Icons.videogame_asset_outlined)),
+    BottomNavigationBarItem(label: 'Stats', icon: Icon(Icons.show_chart)),
+    BottomNavigationBarItem(label: 'Settings', icon: Icon(Icons.settings)),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return BottomNavigationBar(
+      currentIndex: _selectedIndex,
```

Comment:
Czy nie lepiej żeby to była stateless klasa i żeby indeks obecnej zakładki był brany na podstawie BottomNavbarState?

Reply:
Bedzie w cubiie tez